config/middleware.py

```
@@ -5,7 +5,8 @@ class SSODisplayLoggedInCookieMiddleware:
 
     def process_response(self, request, response):
         cookie_value = 'false'
-        if request.user and request.user.is_authenticated():
+        user = getattr(request, 'user', None)
```

when 500 happen before the user is attached to the request. If that happens then the `request has not attribute "user"` error would hide the actual error.